### PR TITLE
Update setup-azure.md

### PR DESCRIPTION
**Pull request by patkearns10**
We figured out an easier way to do this using the azure cli tool. Adding code snippets to help customers do this themselves.

## Checklist

- [ ] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) and [About versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) so my content adheres to these guidelines.
- [ ] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

Adding new pages (delete if not applicable):
- [ ] Add page to `website/sidebars.js`
- [ ] Provide a unique filename for the new page

Removing or renaming existing pages (delete if not applicable):
- [ ] Remove page from `website/sidebars.js`
- [ ] Add an entry `website/static/_redirects`
- [ ] [Ran link testing](https://github.com/dbt-labs/docs.getdbt.com#running-the-cypress-tests-locally) to update the links that point to the deleted page

### CODE REVIEW

File: website/docs/docs/cloud/git/setup-azure.md [hunk 1]
```diff
@@ -117,7 +117,7 @@ A service user account must have the following Azure DevOps permissions for all
 
 \* Note: **DeleteSubscriptions** permission might be included in **EditSubscriptions** depending on your version of Azure.
 
-Some of these permissions are only accessible via the Azure DevOps API, for which documentation can be found [here](https://docs.microsoft.com/en-us/azure/devops/organizations/security/namespace-reference?view=azure-devops). We’ve also detailed more information on Azure DevOps API usage below to help accelerate the set up. Alternatively, you can use the Azure DevOps UI to enable permissions, but you cannot get the least permissioned set.
+Some of these permissions are only accessible via the Azure DevOps API or CLI, for which documentation can be found [here](https://docs.microsoft.com/en-us/azure/devops/organizations/security/namespace-reference?view=azure-devops) and [here](https://learn.microsoft.com/en-us/cli/azure/devops?view=azure-cli-latest) respectively. We’ve also detailed more information on Azure DevOps API usage below to help accelerate the set up. Alternatively, you can use the Azure DevOps UI to enable permissions, but you cannot get the least permissioned set.
```
Comment (mirnawong1) [inline website/docs/docs/cloud/git/setup-azure.md]: shortened text and added links to words

```suggestion
Some of these permissions are only accessible via the [Azure DevOps API](https://docs.microsoft.com/en-us/azure/devops/organizations/security/namespace-reference?view=azure-devops) or [CLI](https://learn.microsoft.com/en-us/cli/azure/devops?view=azure-cli-latest). We’ve also detailed more information on Azure DevOps API usage below to help accelerate the setup. Alternatively, you can use the Azure DevOps UI to enable permissions, but you cannot get the least permissioned set.
```

**Review by mirnawong1 — APPROVED** [review]
hey @patkearns10 great shout and i've approved this for you to merge! let me know if you'd like me to merge for you!

**Comment by patkearns10** [comment]
@mirnawong1  - looks like I dont have authorization to merge. Feel free!

**Comment by mirnawong1** [comment]
Oh gotcha, I think we made changes to this recently and forgot! Thanks so much and will merge!

**Pull request merged**